README.md

```diff
@@ -8,83 +8,44 @@ iOS Playbook 📚
-| Danilo Aliberti         |                                                               |
-| David Rodrigues         | [@dmcrodrigues](https://twitter.com/dmcrodrigues)             |
-| Diego Petrucci          | [@diegopetrucci](https://twitter.com/diegopetrucci)           |
-| Emese Toth              | [@emeseuk](https://github.com/emeseuk)           |
-| Giorgos Tsiapaliokas    | [@gtsiap](https://github.com/gtsiap)                          |
-| Ilya Puchka             | [@ilyapuchka](https://twitter.com/ilyapuchka)                 |
-| James Birtwell          | [@jimmybee](https://github.com/jimmybee)
-| Jason Dobo              | [@jasondobo](https://github.com/jasondobo)                    |
-| João Pereira            | [@NSMyself](https://twitter.com/nsmyself)                     |
-| Joshua Simmons          | [@j531](https://github.com/j531)                              |
-| Julien Ducret           | [@brocoo](https://github.com/brocoo)                          |
-| Konrad Muchowicz        | [@konrad-em](https://github.com/konrad-em)                    |
-| Martin Nygren           |                                                               |
-| Michael Brown           | [@mluisbrown](https://twitter.com/mluisbrown)                 |
-| Michał Kwiecień         | [@kwiecien_co](https://twitter.com/kwiecien_co)               |
-| Nicola Di Pol           | [@nicola.dipol](https://github.com/horothesun)                |
-| Olivier Halligon        | [@AliSoftware](https://twitter.com/aligatr)                   |
-| Rui Peres               | [@peres](https://twitter.com/peres)                           |
-| Sergey Shulga           | [@SergDort](https://twitter.com/SergDort)                     |
-| Simon Cass              | [@codercass](https://twitter.com/codercass)                   |
-| Jakub Tomanik           | [@jtomanik](https://github.com/jtomanik)                      |
-| Viorel Mihalache        | [@viorelMO](https://twitter.com/viorelMO)                     |
-| Witold Skibniewski      |                                                               |
-| Yasuhiro Inami          | [@inamiy](https://twitter.com/inamiy)                         |
-| Yuri Karabatov          | [@karabatov](https://twitter.com/karabatov)                   |
-| Greg Bryant             |                                                               |
-| Sam Francis             |                                                               |
-| Patrick Westmeyer       |                                                               |
-| Daniel Spindelbauer     | [@sdaniel55](https://twitter.com/sdaniel55)                   |
-
-
-## 2. Which squad each individual belongs to, and its availability.
-
-By definition, members work on their respective squad, although they are free to work in different squads if the work load justifies it.
+## 1. Who are we? 
 
+==================================
```

Comment:
Not sure what you expected this to render like, but doesn't seem to render nicely in GitHub.

Maybe you meant to show a horizontal line separator? If so I think it's achieved via `----` in GitHub-Flavored Markdown